docs/content/utilities/documentation/moose_flavored_markdown.md

```diff
@@ -300,3 +302,22 @@ Currently this will only work with Civet CI services.
 ```
 !!! note
     Be sure to follow your !buildstatus extension with an empty new line.
+
+
+---
+
+## Bibliographies
+
+It is possible to include citations using latex commands, the following commands are supported within the markdown.
+
+* `\cite{slaughter2015continuous}`: \cite{slaughter2015continuous}
+* `\citet{wang2014diffusion}`: \citet{wang2014diffusion}
+* `\citep{gaston2015physics}`: \citep{gaston2015physics}
+
+The bibliography style may be set within a page using the latex command
+`\bibliographystyle{unsrt}`. Three styles are currently available: 'unsrt', 'plain', 'alpha', and 'unsrtalpha'.
+
+The references are displayed by using the latex `\bibliography{docs/moose.bib}` command. This command accepts a comma separated list of bibtex files (*.bib) to use to build citations and references. The files specified in this list must be given as a relative path to the root directory (e.g., `~/projects/moose`) of the repository.
+
+\bibliographystyle{unsrt}
+\bibliography{docs/moose.bib}
```

Comment:
I think multiple bibtex files should be allowed, I image that moose.bib would stuff that MOOSE uses or papers about using MOOSE. I also envisioned that there will be additional bib files for the various parts including the modules, tutorials, examples, etc. So, perhaps a "bib" folder is the correct approach.

How about we have docs/bib/*.bib. And we just look for the files in this folder, if they are not there then look for the full path like I am doing here?